packages/ember-glimmer/lib/syntax.ts

```diff
@@ -65,7 +65,11 @@ export function registerMacros(macro: any) {
 export function populateMacros(blocks: any, inlines: any) {
   inlines.add('outlet', outletMacro);
   inlines.add('component', inlineComponentMacro);
-  inlines.add('render', renderMacro);
+
+  if (ENV._ENABLE_RENDER_SUPPORT === true) {
```

Comment:
I think this `ENV` is not imported (thats what one of the test failures indicated at least)..

Reply:
@thoov - I think fixing this will fix CI.

Reply:
@rwjblue Yep that fixed it. Should be good to go now